lib/Debian/Debhelper/Buildsystem/dh_virtualenv.pm

```diff
@@ -38,9 +38,9 @@ sub get_install_root {
 sub get_venv_builddir {
     my $this = shift;
     my $builddir = $this->get_builddir();
-    my $sourcepackage = $this->sourcepackage();
+    my $virtualenv_name = $ENV{DH_VIRTUALENV_INSTALL_SUFFIX} || $this->sourcepackage();
```

Comment:
Ah, scratch that, just realised I hadn't merged the previous one and indeed this PR fixes a bug in that, nice catch.